applications/openshift/master/file_owner_etcd_member/rule.yml

```diff
@@ -2,14 +2,15 @@ documentation_complete: true
 
 prodtype: ocp4
 
-title: 'Verify User Who Owns The ETCD Member Pod Specification File'
+title: 'Verify User Who Owns The etcd Member Pod Specification File'
 
-description: '{{{ describe_file_owner(file="/etc/kubernetes/static-pod-resources/etcd-pod-.*/etcd-pod.yaml", owner="root") }}}'
+description: '{{{ describe_file_owner(file="/etc/kubernetes/static-pod-resources/etcd-pod-*/etcd-pod.yaml", owner="root") }}}'
 
 rationale: |-
-    The ETCD specification file contains information about the configuration of the
-    OpenShift scheduler that is configured on the system. Protection of this file is
-    critical for OpenShift security.
+    If the etcd specification file contains the configuration of
+    the etcd highly-available key-value store which Kubernetes uses for
+    persistent storage of all of its REST API object. Protection of this
+    file is critical for OpenShift security.
```

Comment:
Check each instances of this rationale. The english is off.

Reply:
Please check this version, I more or less reused the rationale from CIS with the upstream kubernetes specific path removed. If would like this version to be improved, I would appreciate a suggestion.